### DISCUSSION TITLE
Block new time entry after a new month has started

### DISCUSSION THREAD

**Discussion by bjinthahouse** [kimai/kimai#4333] [General] [answered]
Hi all,

we're using Kimai in Version 2.0.33 and we track all project related times in it. We are having issues with employees and also external service providers who track times too far in the past. This is especially problematic, if people add new time entries after the invoice has been sent to the customer. So we are looking to block time tracking in the past AFTER the end of the month. Is there a setting I can use, or will this be a feature request?

Cheers!

**Comment by kevinpapst** [comment, marked as answer]
Hey 👋 

Check the lockdown mode: https://www.kimai.org/documentation/configurations.html#lockdown-period

**Reply by kevinpapst** [reply]
Locking down month per user is also possible when using the Working Contract plugin.

**Reply by bjinthahouse** [reply]
Awesome - Thank you very much! That was exactly what I was looking for! 😎
